cdap-docs/developers-manual/source/building-blocks/workflows.rst

```diff
@@ -185,6 +185,89 @@ The diagram for this code would be:
    :width: 8in
    :align: center
 
+Conditional Node
```

Comment:
Also, in the conditional nodes section, we should perhaps upper case D, A, and G and add the acronym (DAG), since its very well-known